web/src/notifications.js

```diff
@@ -609,6 +631,11 @@ export function notify_local_mixes(messages, need_user_to_scroll) {
             continue;
         }
 
+        const muted_narrow = get_muted_narrow(message);
+        if (muted_narrow) {
+            notify_unmute(muted_narrow, message.stream_id, message.topic);
+        }
+
```

Comment:
Just to make sure I understand -- you move this out of `get_local_notify_mix_reason` because we actually want to show it even if the message is visible in the current feed, which would cause `get_local_notify_mix_reason` to exit early?

I think that's reasonable but it looks like this doesn't `continue` so we might send multiple banners in some cases? I assume that's intentional; if so, we should write a comment at the end of that `if` block explaining that explicit decision and the reasoning behind that, if so.

Reply:
Yes, I kept this separate from `get_local_notify_mix_reason` to be able to show the unmute banner as well as any other banner simultaneously for a single message when needed. 

This decision was actually specified in the issue #24246. Quoting it:
>  If sending from outside of narrow, show the regular "outside of narrow" banner above the muted topic banner:

So far in my testing, the banner to switch to the narrow a message is sent to, and the banner to scroll down to view it, have accompanied this unmute banner. There can be at most 1 more banner in addition to the unmute banner. I'll add this in a comment

Reply:
Added the comment